runtime/doc/various.txt

```diff
@@ -208,13 +208,20 @@ g8			Print the hex values of the bytes used in the
 :sh[ell]		Removed. |vim-differences| {Nvim}
 
 						  *:terminal* *:te*
+:te[rminal][!]		Open an interactive shell in a |terminal-emulator|.
+			Use the shell specified in 'shell'.
+
+			The new terminal buffer takes over the current window
+			(like |:edit|).
+
+						  *:terminal_c*
 :te[rminal][!] {cmd}	Execute {cmd} with 'shell' in a |terminal-emulator|
-                        buffer.  Equivalent to: >
```

Comment:
Can't we just add brackets around `{cmd}`? 

    :te[rminal][!] [{cmd}]	Execute {cmd} with 'shell' in a |terminal-emulator|

Also like @mhinz implied, the "take over" is already implied by the words "in a ~~new~~ terminal buffer".  (We should add "new" to that sentence.) 

Users presumably are not surprised when `:edit` "takes over" a window so they should not be surprised by `:terminal`. The reason people _are_ surprised by `:terminal` is because they expect it to behave like `:!`. And IMO `:terminal` _should_ split the window, but it might be too late to change that behavior now....